optimism/src/main.rs

```diff
@@ -185,18 +194,29 @@ pub fn main() -> ExitCode {
 
     {
         // MIPS
-        let mips_proof = proof::prove::<MIPS_COLUMNS, _, OpeningProof, BaseSponge, ScalarSponge>(
-            domain,
-            &srs,
-            mips_folded_witness,
-        );
+        // TODO: use actual constraints, not just an empty vector
+        // FIXME: this means create separate MIPS witnesses and prove the corresponding constraints for each
+        let mips_result = prove::<
+            _,
+            OpeningProof,
+            BaseSponge,
+            ScalarSponge,
+            Column,
+            _,
+            MIPS_COLUMNS,
+            LookupTableIDs,
+        >(domain, &srs, &vec![], mips_folded_witness, &mut rng);
```

Comment:
Now the main uses the generic prover/verifier, but in this PR I am not splitting into separate witnesses for different constraint types. Also, the dummy folding we are performing breaks the witness, so real constraints would not satisfy. For that reason, I am just passing the empty list of constraints so that the test passes. 

In a future PR, I will adapt the `main` to run the constraints. Probably, for that I will just use a dummier fold which just keeps the last instance instead of a random combination.